api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -426,6 +432,39 @@ public void testGetCohortsInWorkspace() throws Exception {
     assertThat(cohorts.size()).isEqualTo(2);
   }
 
+  @Test
+  public void testCreateCohortAccessLevelOwner() {
+    Cohort saved = createTestCohort(WorkspaceAccessLevel.OWNER);
```

Comment:
It's not clear that your testing the controller#createCohort method with the logic buried in createTestCohort method. I have to drill into this method to figure out what is going on. By the name of the method I'm assuming it just creates a test Cohort object, but in reality it is doing much more(stubbing workspace and workspace acl and calling a controller method). I prefer giving this a more descriptive name or splitting it into more than one method, something like stubWorkspaceAndAcl(), createTestCohort() and callControllerCreateCohort()

Reply:
updated code to address this issue